smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/model/SymbolBuilder.kt

```diff
@@ -108,6 +109,17 @@ fun Symbol.isOptional(): Boolean {
     return this.getProperty("isOptional").orElse(false) as Boolean
 }
 
+fun Symbol.isInternalSPI(): Boolean {
+    return this.getProperty("isInternalSPI").orElse(false) as Boolean
+}
+
+fun Symbol.toInternalSPI(kind: SwiftKind): Symbol {
+    return this.toBuilder()
+        .putProperty("isInternalSPI", true)
+        .putProperty("kind", kind.kind)
+        .build()
+}
+
```

Comment:
Utility methods to add properties to a `Symbol` for SPI import logic in `SwiftWriter`.